packages/blitz-auth/src/client/index.tsx

```diff
@@ -248,6 +248,11 @@ export function getAuthValues<TProps = any>(
 function withBlitzAuthPlugin<TProps = any>(Page: ComponentType<TProps> | BlitzPage<TProps>) {
   const AuthRoot = (props: ComponentProps<any>) => {
     useSession({suspense: false})
+    const [mounted, setMounted] = useState<boolean>(false)
+
+    useEffect(() => {
+      setMounted(true)
+    }, [])
```

Comment:
Probably also need a clean up function:
```suggestion
    useEffect(() => {
      setMounted(true)
      
      return () => setMounted(false)
    }, [])
```

Reply:
cleanup function not needed. Cleanup function only needed for memory management in cases where you add event handlers or subscriptions